Detectors/ITSMFT/ITS/macros/test/CheckDigits.C

```diff
@@ -88,7 +88,7 @@ void CheckDigits(Int_t nEvents = 10, TString mcEngine = "TGeant3") {
 	for (Int_t i=0; i<nh; i++) {
 	  Hit *p=(Hit *)hitArr.UncheckedAt(i);
 	  if (p->GetDetectorID() != chipID) continue; 
-	  if (p->GetTrackID() != lab) continue;
+	  if (p->GetTrackID() != trID) continue;
```

Comment:
@iouribelikov : thanks for fixes I forgot to do. But this one was fixed (+ extra info added to control ntuple) in my PR616: https://github.com/AliceO2Group/AliceO2/pull/616/commits/18bd66d881a432dc53bf8af79ce0c9ced297a8c0